main.cpp

```diff
@@ -0,0 +1,112 @@
+#include <string>
+#include <VSScript4.h>
+#include <elements.hpp>
+
+using namespace cycfi::elements;
+
+int main(int argc, char** argv)
+{
+    try {
+        // Init variables
+        int default_frame_number = 0;
+        const char *script_file_path{[&] {
+            return argv[1] ? argv[1] : throw std::runtime_error{"VSPreview error: Script file not specified"};
+        }()};
+        int frame_number{[&] {
+            if (!argv[2]) {
+                return default_frame_number;
+            }
+            std::stringstream str{argv[2]};
+            int number;
+            str >> number;
```

Comment:
use `std::stoi` instead. Ideally we should use a library like cxxopts to parse `argv`